src/nyc_trees/apps/survey/urls/census_admin.py

```diff
@@ -18,6 +20,10 @@
 # These URLs have the prefix 'census_admin/'
 urlpatterns = patterns(
     '',
+    url(r'^territory/$',
+        census_admin_do(route(GET=r.admin_territory_page)),
```

Comment:
I agree with @kdeloach here, we don't want to double decorate things.

Also, we should remove the existing `admin_blockface_page` view/route/url since we're not using them.